sqlmesh/core/snapshot/evaluator.py

```diff
@@ -652,6 +652,8 @@ def _create_snapshot(
             **common_render_kwargs,
             # Refers to self as non-deployable to successfully create self-referential tables / views.
             deployability_index=deployability_index.with_non_deployable(snapshot),
+            # However, it can still be useful to know if the snapshot was actually deployable
+            is_snapshot_deployable=deployability_index.is_deployable(snapshot),
```

Comment:
The deployability index is already in the render kwargs which is why I put `is_snapshot_deployable` right next to it

Reply:
Moved to `kwargs`